src/microengine_utils/scanner.py

```diff
@@ -25,6 +26,13 @@
 from .errors import BaseScanError, CalledProcessScanError
 
 
+class ScanResult(BaseModel):
+    bit: bool
+    verdict: bool
+    confidence: float = 1.0
+    metadata: Verdict = Field(default_factory=lambda: Verdict().set_malware_family('').json())
+
+
```

Comment:
This can be a bit confusing here as it is. There is another similar-named class in the same package in another module. Which one an engine author should be using? What is the difference?

What about adding some docstring or even a bare comment pointing out that this is just an adapter for a past version of a helper library? Or something that just clarifies this duplicity reason.